### ISSUE TITLE
Don't check indentation within code blocks

### ISSUE THREAD

**Issue by iansan5653** [gajus/eslint-plugin-jsdoc#334]
Currently, if `jsdoc/check-indentation` is set and a markdown code block is included that contains indentation, the comment is considered to be an error because there is indentation in the comment. However, code blocks are expected to be properly indented for display, so this rule should either not check within blocks, or it should have an option to ignore code blocks.

Example:
```javascript
/**
 * This is a class that injects some HTML elements.
 * The injected elements can be styled as follows:
 * ```xml
 * <container>
 *  <element class="readout__gauge gauge">
 *    <element class="gauge__name">{Caption}</element>
 *    <element class="gauge__value">{Value}</element>
 *  </element>
 *  <element class="readout__gauge gauge">
 *    <element class="gauge__name">{Caption}</element>
 *    <element class="gauge__value">{Value}</element>
 *  </element>
 * </container>
 * ```
 */
```
